### PR TITLE
Upgrade prometheus-operator

### PR DESCRIPTION
**Pull request by gracedo**
Upgrade to v5.19.1

### CODE REVIEW

**Comment by shaneutt** [comment]
Looks like this is `ready`, but blocked on https://github.com/mesosphere/charts/pull/144

File: templates/prometheus.yaml
Comment (shaneutt) [inline templates/prometheus.yaml]: Let's just make sure we bump this back to mesosphere when we're unblocked before merge :+1:

**Review by shaneutt — APPROVED** [review]

**Comment by hectorj2f** [comment]
@gracedo what is the state of this PR ?

**Review by s12chung — APPROVED** [review]

**Review by s12chung — CHANGES_REQUESTED** [review]
ah, keep forgetting, please update the url in line 22, the url should point to the values of the chart. the current url is a bug pointing to the wrong chart and it should be updated when the values.yaml change on the chart.

**Comment by shaneutt** [comment]
@s12chung I think we're just waiting on your re-review to :heavy_check_mark: and merge this one

**Review by s12chung — APPROVED** [review]

**Comment by s12chung** [comment]
my bad. forgot about this one. ✅

**Pull request merged**
